app/src/scripts/API/ModelAPIs/AuthAPI.ts

```diff
@@ -20,15 +21,17 @@ export default class AuthAPI {
      * @returns A promise that is either a pair of token or an error.
      */
     static loginRequest(login: string, password: string): Promise<APIResponse<TokenPair>> {
-        const body = {
-            username: login,
-            password: password,
-        }
+        // Créer les données au format x-www-form-urlencoded
+        // Bien utiliser URLSearchParams pour éviter les problèmes de caractères spéciaux
+        const formData = new URLSearchParams();
+        formData.append('username', login);
+        formData.append('password', password);
```

Comment:
Good call, since this route needs to use an URL_FORM_ENCODED body due to OAuth Standard.